packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -581,10 +551,8 @@ contract Connext is
       });
     }
 
-    // Pay metatx relayer
-    // NOTE: if this is done *without* fast liquidity, router will be address(0) and the relayer
-    // will always be paid
-    _handleRelayerFees(_transferId, _args.routers, _args.feePercentage, _args.relayerSignature);
+    // Set the relayer for this transaction to allow for future claim
+    transferRelayer[_transferId] = msg.sender;
```

Comment:
`_handleRelayerFees` previously checked the router's `relayerSignature`... removing this means that the relayer can now call this execute fn using any router. There's no longer any security behind execute except the whitelist with the relayers.

While we aren't in prod yet so this isn't a real concern, I'm wondering if we should go ahead and knock this essential piece of tech-debt out of the way among the relayer fee changes here.

The essential piece I'm referring to: having a *different* signature from the router that makes this tx securely metatx-able. The signature digest should be something like `[transferId, router fee %, auction round number]`

We can write the check in a sep. branch, but personally I think it's worth considering it a blocker for getting this in. I'd *like* to proceed with that security mechanism in place lol.